api/node/Dockerfile

```diff
@@ -4,8 +4,11 @@ WORKDIR /usr/app
 COPY .yarn ./.yarn
 COPY package*.json yarn.lock .yarnrc.yml ./
 
+RUN apt-get update
+RUN apt-get install -y build-essential libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev
```

Comment:
@Fosol - I like your idea to make a Dockerfile.mac or something of the likes - I seem to be getting an error from this above line on Windows

Reply:
Seems to be the only thing failing though